src/sentry/tagstore/snuba/backend.py

```diff
@@ -534,27 +534,16 @@ def get_group_tag_keys_and_top_values(
         return keys_with_counts
 
     def __get_release(self, project_id, group_id, first=True):
```

Comment:
I don't think this belongs in tagstore anymore, since we're no longer querying Snuba here. Can we move this into `group.get_(first|last)_release`?

Nice thing here is that we can change the repair query in `get_first_release` to just use this query as well.